frontend/src/app/App.tsx

```diff
@@ -210,44 +210,47 @@ const App = () => {
                 }
               />
               <Route
-                path="csv/codelookup"
+                path="results/upload/submit"
```

Comment:
changed the order we define routes on this page just for readability - submit first, then guide and code-lookup which essentially fall under submit